Quadriga/src/main/java/edu/asu/spring/quadriga/web/publicwebsite/WebsiteProjectController.java

```diff
@@ -82,15 +116,24 @@ public int compare(IPublicPage o1, IPublicPage o2) {
         Map<String, String> linkToMap = getLinkTargetMap();
         publicPages.forEach(item -> item.setLinkTo(linkToMap.get(item.getLinkTo())));
 
-        model.addAttribute("blocks", publicPages);
+        // Fetch blog entries for a project identified by project unix name
+        List<IProjectBlogEntry> latestProjectBlogEntryList = projectBlogEntryManager.getProjectBlogEntryList(projectId,
```

Comment:
just use ```project.getProjectId()``` here and loose the variable at the top. Or, keep the variable but use it consistently throughout the method. Right now it's mixed.